lindsayreiner/pocketRM
# added seed data, changed name to first and last in schema and type defs, registration validation updates

### To test
- go to the registration page and attempt to submit the form blank and errors should populate to show that all fields are required. 
- Then attempt to add a name that includes anything that is not a letter or ,.'- to see an error to check the spelling of the first name or last name. 
- In the email field, enter anything that is not formatted correctly as an email address and it will provide an error that the email address is invalid. 
- In the password field, if it is not at least 8 characters with at least one letter and one number it will return an error stating that.
